notebooks/official/tensorboard/tensorboard_vertex_pipeline.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>that includes -&gt; including</p><p> </p><p>Add periods after each bullet point.</p><p> </p><p>metrics such as -&gt; metrics, such as</p><p> </p><p>provides: -&gt; provides the following benefits:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>